app/src/main/java/xyz/klinker/blur/launcher3/Launcher.java

```diff
@@ -502,6 +511,15 @@ protected void onCreate(Bundle savedInstanceState) {
         }
     }
 
+  public void showGlobalSearch() {
+        startSearch(null, false, null, true);
+    }
+
+    public void g(){
```

Comment:
your phone might be of samsung or other manufaturer who not use aosp calender. Tell now  if it crashes . showCalendar method is done